examples/point_cloud/point_cloud.cpp

```diff
@@ -131,8 +131,8 @@ uint32_t GenerateXYZTables(float **pp_x_table, float **pp_y_table,
     // Adjust values based on optical center and focal length
     float cx = p_intr_data->cx / row_bin_factor;
     float cy = p_intr_data->cy / col_bin_factor;
-    float fx = p_intr_data->fx / row_bin_factor;
-    float fy = p_intr_data->fy / col_bin_factor;
+    //float fx = p_intr_data->fx / row_bin_factor;
+    //float fy = p_intr_data->fy / col_bin_factor;
```

Comment:
Please don't leave commented code. Delete it if not needed.

Reply:
Solved